realsense_camera/launch/r200_nodelet_modify_params.launch

```diff
@@ -9,11 +9,11 @@
   <!-- These 'arg' tags are just place-holders for passing values from test files.
   The recommended way is to pass the values directly into the 'param' tags. -->
   <arg name="enable_depth"      default="true" />
-  <arg name="enable_ir"         default="true" />
-  <arg name="enable_ir2"        default="true" />
-  <arg name="enable_color"      default="true" />
-  <arg name="enable_pointcloud" default="true" />
-  <arg name="enable_tf"         default="true" />
+  <arg name="enable_ir"         default="false" />
```

Comment:
We should change these in the launch file that calls this launch file.  Also can repurpose the camera frequency to decimate (reduce) the framerate.  That way we can keep the same api interface to the realsense library.